sdcm/cluster.py

```diff
@@ -794,15 +801,18 @@ def _notify_backtrace(self, last):
         for line in output.splitlines():
             self.log.error(line)
 
+    @retrying(n=10, sleep_time=20, allowed_exceptions=NETWORK_EXCEPTIONS, message="Reconnecting")
```

Comment:
why putting the retry here ? this is getting called in a loop, the next time it called might work.
in this case after 10 attempts the thread will fail, and next coredumps might be missed. 
on the other hand it might try uploading infinite times.

meh, since it's network issues, maybe it's not that critical where is the retry located, just that it would happen

Reply:
In order to do perfect error handling here we will have to rewrite remoter.run and probably touch pamiko to make it signaling if command was delivered to the endpoint.
It is out of scope of the task.